src/docs/development/androidx-migration.md

```diff
@@ -62,7 +62,7 @@ To create a new project run:
 flutter create --androidx -t <project-type> <new-project-path>
 ```
 
-### How do I migrate my existing module project to AndroidX?
+### Add to App
 
 If your project is a module and contains a `.android` directory, add the following
```

Comment:
If your Flutter project is a module type for adding to an existing Android app, and contains a ...